### ISSUE TITLE
Some  possible PubMed craziness

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#1194]
I was slightly side tracked trying to  look at taxon ID stuff.

From this page
https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=4896&lvl=3&keep=1&srchmode=1&unlock&lin=s&log_op=lineage_toggle

15 538 PMC publications are  recorded. That's considerably more than we have

So in the list is this

Reconstitution of Targeted Deadenylation by the Ccr4-Not Complex and the YTH Domain Protein Mmi1
James A.W. Stowell, Michael W. Webster, Alexander Kögel, Jana Wolf, Kathryn L. Shelley, Lori A. Passmore
Cell Rep. 2016 Nov 15; 17(8): 1978–1989. Published online 2016 Nov 15. doi: 10.1016/j.celrep.2016.10.066
PMCID: PMC5120349

which in pubmed is PMID: 27851962

and does appear to have pombe experimental data

We also demonstrate that the Schizosaccharomyces pombe RNA-binding protein Mmi1 binds stably to recombinant Ccr4-Not.

and is not in Canto

Others I just don't understand....
PMCID: PMC5101800 is
PMID: 27825301 
is not in canto but has Schizosaccharomyces pombe in the title.

Largely they seem to be nothing to do with pombe at all.

Maybe best strategy is to wait see if the above come in via our usual pipeline soon.....

**Comment by mah11** [comment]
> PMID:27851962 does appear to have pombe experimental data ... and is not in Canto

This one is in Canto, not yet triaged.

> PMID:27825301 is not in canto but has Schizosaccharomyces pombe in the title.

This is in Canto, triaged "Sequence feature or region".

For the broader point, I had a little dig round, and now I think I understand it.

1. The query URL retrieves every PMC entry that has Schizosaccharomyces or Schizosaccharomyces pombe (or both) in its list of taxonomy links.

2. A PMC entry gets tagged for inclusion in the list of links for any taxon that gets mentioned using its scientific name anywhere in the text, including the introduction, references, etc. For example, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5080547/ (which I chose more or less randomly) has "In the fission yeast Schizosaccharomyces pombe, genomic screens identified genes ... " in the intro and one reference with "Schizosaccharomyces pombe" in the title.

From the fact that we don't have 15,000 papers in Canto, I infer that the search criteria it's using are more stringent.


tl;dr: looks like nothing to worry about :)

**Comment by ValWood** [comment]
Phew....thanks!
